pkg/beholder/client.go

```diff
@@ -0,0 +1,305 @@
+package beholder
+
+import (
+	"context"
+	"errors"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	otellog "go.opentelemetry.io/otel/log"
+	otelmetric "go.opentelemetry.io/otel/metric"
+	sdklog "go.opentelemetry.io/otel/sdk/log"
+	sdkmetric "go.opentelemetry.io/otel/sdk/metric"
+	sdkresource "go.opentelemetry.io/otel/sdk/resource"
+	"go.opentelemetry.io/otel/sdk/trace"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	oteltrace "go.opentelemetry.io/otel/trace"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type MessageEmitter interface {
+	// Sends message with bytes and attributes to OTel Collector
+	Emit(ctx context.Context, body []byte, attrKVs ...any) error
+}
+type Client interface {
```

Comment:
What is this interface for? And is it the only reason that `OtelClient` is not named just `Client`? Seems like we could simplify a lot by dropping it.

Reply:
> What is this interface for?
Initially, it was planned to be used for testing/mocking. 
After refactoring otel client constructors do not return the interface but OtelClient struct, so it can be removed. 

Removed, renamed here https://github.com/smartcontractkit/chainlink-common/pull/694/commits/1582cc2709b6da497f708e62971162128c9942d4